src/uzfs_prop.c

```diff
@@ -0,0 +1,41 @@
+#include <zrepl_mgmt.h>
+#include <uzfs_mgmt.h>
+
+/*
+ * This file defines function to set property of uzfs zvol
+ */
+int
+uzfs_zfs_set_prop(const char *name, zprop_source_t source, nvlist_t *list)
```

Comment:
lets set mgmt_conn's related flag here itself as zinfo is already available here, instead of setting in update_zvol_property which should be just aware of zv related things

Reply:
done